src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -22,28 +21,38 @@ class EnrollmentHistory extends React.Component {
     */
 
     // History explanation box
+    let historyExplanationBox;
     const benefitMaybePending =
-      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement > 0) &&
-      enrollmentData.usedEntitlement === 0;
+      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement.months > 0) &&
+      (enrollmentData.usedEntitlement.months === 0 && enrollmentData.usedEntitlement.days === 0);
     const historyMayLookIncorrect = enrollments.length > 0 || benefitMaybePending;
-    const noEnrollmentHistory = enrollments.length === 0 && enrollmentData.usedEntitlement > 0;
-    const historyHeading = noEnrollmentHistory ?
-                           (<h4>You don't have any enrollment history</h4>) :
-                           (<h4>Does something look wrong in your enrollment history?</h4>);
-    const historyExplanation = noEnrollmentHistory ?
-                               (<span>Your enrollment history may not be available if:</span>) :
-                               (<span>Certain enrollments may not be displayed in this history if:</span>);
-    const historyExplanationBox = (historyMayLookIncorrect || noEnrollmentHistory) && (
-      <div className="feature">
-        {historyHeading}
-        {historyExplanation}
-        <ul>
-          <li>Your school made a request to us that's still in process, or</li>
-          <li>You made a request to us that's still in process, or</li>
-          <li>You used or are using your benefit for flight, on-the-job, apprenticeship, or correspondence training</li>
-        </ul>
-      </div>
-    );
+    const noEnrollmentHistory = enrollmentData.usedEntitlement.months === 0 && enrollmentData.usedEntitlement.days === 0;
```

Comment:
Ah, OK...so there are 2 (slightly different) UIs indicating no enrollment history.

1) The one that was here before that checks `enrollments.length === 0 && enrollmentData.usedEntitlement > 0` which I believe we implemented for this mock: https://marvelapp.com/8d6igd9/screen/29281166

2) And this new one, which checks `enrollmentData.usedEntitlement.months === 0 && enrollmentData.usedEntitlement.days === 0` and is listed in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3839

I think we need to verify with @phiden and @rroueche what conditions should be met for https://marvelapp.com/8d6igd9/screen/29281166 to be displayed. It was originally in this ticket https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3297.